src/main/java/org/prebid/server/floors/BasicPriceFloorAdjuster.java

```diff
@@ -34,6 +44,21 @@ private static ExtRequestBidadjustmentfactors extractBidadjustmentfactors(BidReq
         return ObjectUtil.getIfNotNull(extPrebid, ExtRequestPrebid::getBidadjustmentfactors);
     }
 
+    private static ImpMediaType chooseMediaType(ExtRequestBidadjustmentfactors extBidadjustmentfactors, String bidder) {
+        final TreeMap<BigDecimal, ImpMediaType> factorToMediatype = new TreeMap<>();
+
+        for (ImpMediaType mediaType : extBidadjustmentfactors.getMediatypes().keySet()) {
```

Comment:
Seems extBidadjustmentfactors.getMediatypes() can be null let's add tests for it